minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -259,6 +259,7 @@ private void startMaster(TachyonConf testConf) throws IOException {
     if (ufs.getClass().getSimpleName().equals("LocalMiniDFSCluster")) {
       testConf.set(Constants.UNDERFS_ADDRESS, ufs.getUnderFilesystemAddress() + mTachyonHome);
       MasterContext.reset(testConf);
+      mMasterConf = testConf;
```

Comment:
I am a bit confused by the code block:
```java
MasterContext.reset(testConf);
mMasterConf = testConf;
```
By `MasterContext.reset(testConf);` , the previous conf in master context will be lost, and it is a bit different from `testConf` here because when master starts, we update a bunch stuff inside the conf.